modules/serverless-connect-sink-source-odc.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+//  * serverless/channels/connect-channels-event-source.adoc
+// * serverless/event_sources/srvls-connecting-source-sinks.adoc
+
+[id="serverless-connect-sink-source-odc_{context}"]
+= Connect an event source to a sink using the Developer perspective
+
+You can create multiple event source types in {product-title} that can be connected to sinks.
+
+A sink is an addressable resource that can receive incoming events from other Knative Eventing resources. Knative services, channels, and brokers are all examples of sinks.
+// Add functions as services to the list for tech preview
+
+.Prerequisites
+
+To connect an event source to a sink using the *Developer* perspective, ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
```

Comment:
I think there is an extra ',' here... but I'll leave it up to you..
```suggestion
* The {ServerlessOperatorName}, Knative Serving and Knative Eventing are installed on your {product-title} cluster.
```

Reply:
We generally use Oxford commas in OCP docs AFAIK so I'll leave it in